gif.BUILD

```diff
@@ -15,13 +15,49 @@ HEADERS = [
     "gif_lib_private.h",
 ]
 
+config_setting(
+        name = "windows",
+        values = {
+            "cpu": "x64_windows_msvc",
+        },
+        visibility = ["//visibility:public"],
+)
+
 prefix_dir = "giflib-5.1.4/lib"
+prefix_dir_windows = "windows/giflib-5.1.4/lib"
+
+genrule(
+  name = "srcs_without_unistd",
+  srcs = [prefix_dir + "/" + source for source in SOURCES],
+  outs = [prefix_dir_windows + "/" + source for source in SOURCES],
+  cmd = "for f in $(SRCS); do " +
+        "  sed 's/#include <unistd.h>//g' $$f > $(@D)/%s/$$(basename $$f);" % prefix_dir_windows +
```

Comment:
@martinwicke: IIUC running Bazel on Windows currently requires msys, but the generated binaries would not depend on it (since they're compiled using MSVC).